### PR TITLE
Created top male names in Turkey

### PR DESCRIPTION
**Pull request by Alisha-786**
I have added a text file of top male names in Turkey. Kindly check and merge my pull request.

### CODE REVIEW

**Comment by kkrypt0nn** [comment]
Could you please do a single PR and add them to this PR if you have more of these, thanks.

File: wordlists/names/top_male_names_turkish.txt [hunk 1]
```diff
@@ -0,0 +1,873 @@
+Adem
+Adem
```
Comment (kkrypt0nn) [inline wordlists/names/top_male_names_turkish.txt:2]: Duplicates

File: wordlists/names/top_male_names_turkish.txt [hunk 2]
```diff
@@ -0,0 +1,873 @@
...
+Ahmet
+Ahmet
+Ahmet
```
Comment (kkrypt0nn) [inline wordlists/names/top_male_names_turkish.txt:5]: Duplicates

File: wordlists/names/top_male_names_turkish.txt [hunk 3]
```diff
@@ -0,0 +1,873 @@
...
+Ali
+Ali
+Alp
+Alparslan
+Alper
+Alper
+Alperen
+Alperen
+Alperen
+Alperen
```
Comment (kkrypt0nn) [inline wordlists/names/top_male_names_turkish.txt:17]: Duplicates

File: wordlists/names/top_male_names_turkish.txt [hunk 4]
```diff
@@ -0,0 +1,873 @@
+Adem
+Adem
+Ahmet
+Ahmet
+Ahmet
+Akın
+Ali
+Ali
+Ali
```
Comment (kkrypt0nn) [inline wordlists/names/top_male_names_turkish.txt:9]: Duplicates

File: wordlists/names/top_male_names_turkish.txt [hunk 5]
```diff
@@ -0,0 +1,873 @@
...
+Alp
+Alparslan
+Alper
+Alper
```
Comment (kkrypt0nn) [inline wordlists/names/top_male_names_turkish.txt:13]: Duplicates

**Review by kkrypt0nn — CHANGES_REQUESTED** [review]
See my comment above, and remove **all** the duplicates, I've only marked a few.

File: wordlists/names/top_male_names_turkish.txt (hunk 5 above)
Comment (Alisha-786) [reply wordlists/names/top_male_names_turkish.txt:13]: ok I am removing all the duplicates

Comment (Alisha-786) [reply wordlists/names/top_male_names_turkish.txt]: Sorry for the inconvenience. I have updated and there are no duplicates.

**Review by kkrypt0nn — CHANGES_REQUESTED** [review]
Please [sort them](https://appdevtools.com/sort-lines).

If you have any other of these kind wordlists, please add them now to **this** PR. Otherwise I'll think about what to do with your next PR.

**Comment by Alisha-786** [comment]
I have sorted them. Please check.
And this is the last PR of these kind of wordlist.

**Comment by kkrypt0nn** [comment]
You've removed two names instead of sorting them. Use the website I've linked in my response to sort them correctly.

**Comment by Alisha-786** [comment]
Now is it correct?

**Review by kkrypt0nn — APPROVED** [review]
Thanks for sorting them! 👍

**Comment by Alisha-786** [comment]
Sorry for the inconvenience & Thank you very much for merging my PR.

**Pull request merged**